remotes/push.go

```diff
@@ -235,3 +245,75 @@ func pushBundleConfigDescriptor(ctx context.Context, name string, resolver remot
 	}
 	return descriptor, nil
 }
+
+func pushUnderTag(ctx context.Context, imageClient client.ImageAPIClient, image *bundle.BaseImage, targetRef reference.Named) error {
+	ref := image.Digest
```

Comment:
I think we shouldn't use that `content digest` here, as explained by @glyn https://github.com/deislabs/cnab-go/issues/145#issuecomment-541554027 at least until the specifications defines clearly what's in this field.

Reply:
Oh, I think this is the cause for https://github.com/docker/cnab-to-oci/pull/76#issuecomment-542354769